presto-verifier/src/main/java/com/facebook/presto/verifier/framework/DataVerification.java

```diff
@@ -71,6 +72,13 @@ protected QueryObjectBundle getQueryRewrite(ClusterType clusterType)
         return queryRewriter.rewriteQuery(getSourceQuery().getQuery(clusterType), clusterType);
     }
 
+    @Override
+    protected void updateQueryInfoWithQueryBundle(QueryInfo.Builder queryInfo, Optional<QueryObjectBundle> queryBundle)
+    {
+        super.updateQueryInfoWithQueryBundle(queryInfo, queryBundle);
+        queryInfo.setOutputTableName(queryBundle.map(queryObjectBundle -> queryObjectBundle.getObjectName().toString()));
```

Comment:
nit: prefer to chain `.map()` with function reference when possible
```
queryBundle.map(QueryObjectBundle::getObjectName).map(QualifiedName::toString);
```

Reply:
Thanks. Resolved.